$OpenBSD: patch-src_tls_c,v 1.1 2017/02/01 22:00:12 juanfra Exp $

net.h is deprecated. This is just a workaround for our outdated version of
mbedtls. Unneeded with mbedtls 2.4 or later.

--- src/tls.c.orig	Wed Feb  1 22:38:36 2017
+++ src/tls.c	Wed Feb  1 22:39:31 2017
@@ -31,7 +31,7 @@
 #include "mbedtls/ssl_cache.h"
 #include "mbedtls/error.h"
 #include "mbedtls/version.h"
-#include "mbedtls/net_sockets.h"
+#include "mbedtls/net.h"
 #include "mbedtls/ssl.h"
 #include "mbedtls/x509.h"
 #ifdef ENABLE_DEBUG
